src/ft_token_receiver.rs

```diff
@@ -55,6 +57,20 @@ impl FungibleTokenReceiver for Contract {
                 draft_group.fund(&sender_id);
                 self.draft_groups.insert(&draft_group_id as _, &draft_group);
                 log!("Funded draft group {}", draft_group_id);
+
+                if funding.try_convert.unwrap_or(false) {
+                    // Using remaining gas to try convert drafts, not waiting for results
+                    if let Some(remaining_gas) = env::prepaid_gas().checked_sub(GAS_FT_ON_TRANSFER) {
```

Comment:
I suggest use `env::used_gas() + GAS_EXT_CALL_COST` instead of `GAS_FT_ON_TRANSFER`
`GAS_EXT_CALL_COST` should be around 5TGas

Basically you'll need gas to schedule a promise `ext_self::convert_drafts` and pass all the `remaining gas`.

Because currently it assumes that the `used gas + promise call <=  GAS_FT_ON_TRANSFER`